Fundamentals of backend development/createTables.pgsql

```diff
@@ -0,0 +1,49 @@
+
+
+CREATE TABLE IF NOT EXISTS Restaurants (
+      ID_Restaurant SERIAL PRIMARY KEY
```

Comment:
nothing wrong with the names of the tables or columns but in practice i frequently see lowercase for table and column names, but again just a convention thing not right or wrong